src/directives/routeOnClick.js

```diff
@@ -57,7 +57,7 @@ function routeOnClickFactory (Route, $location, $window, $timeout) {
       }
 
       function html5TheUrl(url) {
-        return Route.isHtml5ModeEnabled() ? url : `#${url}`;
+        return Route.isHtml5ModeEnabled() || url.includes('http') ? url : `#${url}`;
```

Comment:
url.includes('http') is probably not ideal as you could easily have that string somewhere inside the url. Try something like this instead:

`url.test(/^(https?:){0,1}\/\/.+/)`

This will work with http://, https:// and relative protocol // urls